config/deploy.rb

```diff
@@ -0,0 +1,55 @@
+# config valid only for current version of Capistrano
+# bundle exec cap production deploy
+# bundle exec cap production rake task=db:seed
+lock '3.4.0'
+# set :use_sudo, true
+set :repo_url, 'https://github.com/pustovalov/app.git'
+set :branch, 'namespace'
+set :deploy_to, '/home/deploy/applications/app'
+# set :puma_threads, [4, 16]
+# set :puma_workers, 0
+
+set :stages, %w(staging production)
+set :default_stage, "production"
+
+set :log_level, :info
+set :linked_files, %w{config/database.yml config/settings.yml}
+set :linked_dirs, %w{bin log tmp/pids tmp/cache tmp/sockets vendor/bundle public/uploads}
+
+set :rbenv_type, :user
+set :rbenv_ruby, '2.2.4'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.